source/manual/data-gov-uk-operations.html.md

```diff
@@ -69,56 +37,34 @@ The sync is normally done automatically using Sidekiq Scheduler. There may be ti
+kubectl exec deploy/datagovuk-publish -n datagovuk -- bash -c 'psql $DATABASE_URL -c "DROP OWNED BY ckan;"'
 
-## make sure the database is empty
-/tmp/lifecycle/launcher /home/vcap/app 'rails db:drop db:setup' ''
+# NOTE - ignore the server restart warnings after setting up the database
+kubectl exec deploy/datagovuk-publish -n datagovuk -- bin/setup
 
 ## make sure the index is setup
-/tmp/lifecycle/launcher /home/vcap/app 'rails search:reindex' ''
 
-## sync datasets or update orgs
-/tmp/lifecycle/launcher /home/vcap/app 'rails runner CKAN::V26::PackageSyncWorker.new.perform' ''
-/tmp/lifecycle/launcher /home/vcap/app 'rails runner CKAN::V26::CKANOrgSyncWorker.new.perform' ''
+kubectl exec deploy/datagovuk-publish -n datagovuk -- rails search:reindex
+
+## update orgs or datasets
+
+kubectl exec deploy/datagovuk-publish -n datagovuk -- rails runner CKAN::V26::CKANOrgSyncWorker.new.perform
+kubectl exec deploy/datagovuk-publish -n datagovuk -- rails runner CKAN::V26::PackageSyncWorker.new.perform
 ```
 
-Now run `bundle exec sidekiq` and `rails s` and monitor the resulting jobs in the [Sidekiq Web UI](/manual/data-gov-uk-monitoring.html#sidekiq-publish).
+Now run `kubectl exec deploy/datagovuk-publish -n datagovuk -- bundle exec sidekiq` and see the logs.
 
 #### Updating Zendesk password for Find
 
 The contact form on Find uses the Zendesk API to create new tickets.  If the account password expires, it will need updating in secrets on the PaaS.
 
 The following example is for staging.
 
-Get the existing credentials (username and password):
-
-```
-cf ssh find-data-beta-staging
-echo $VCAP_SERVICES
-```
-
-Log into the Zendesk using the username and password obtained from the previous step.  Reset the password through the UI.
-
-Update the password in secrets using [uups](http://cli.cloudfoundry.org/en-US/cf/update-user-provided-service.html):
+Get the existing credentials (username and password) from AWS secrets manager under `govuk/dgu/datagovuk`:
 
-```
-cf uups find-staging-secrets -p '{"key1": "value1", "key2": "value2"}'
-```
+Log into the Zendesk using the username and password obtained from the previous step.  Reset the password through the UI and copy the password to the clipboard.
 
-> Updating the credentials will overwrite any existing credentials, not just the one you are updating.  Therefore you must copy the entirety of the credentials from `echo $VCAP_SERVICES`, update the password then paste this into the `uups` command.  Otherwise other credentials will be lost.
+Update the password in AWS secrets manager.
 
-Restage the application to update the environment variables:
-
-```
-cf restage find-data-beta-staging
-```
-
-Verify the new password is being used:
-
-```
-cf ssh find-data-beta-staging
-echo $VCAP_SERVICES
-```
+When you make changes to secrets they will not be automatically picked up by the apps so you have to run this command to pick up the new secrets `kubectl rollout restart deploy -n datagovuk`
```

Comment:
Think we might have [reloader](https://github.com/stakater/Reloader) everywhere now.

Reply:
does that mean that this no longer needs to be run manually? I'll take the line out then